### PR TITLE
feat: add storybook [MDS-1442]

### PR DESCRIPTION
**Pull request by robb91-dev**

### CODE REVIEW

File: docs/.eslintrc.json [hunk 1]
```diff
@@ -1,4 +1,10 @@
...
-}
+  "plugins": [
+    "prettier"
+  ],
+  "extends": [
+    "next/core-web-vitals",
+    "prettier",
+    "plugin:storybook/recommended"
+  ]
+}
```
Comment (MarioGranada) [inline docs/.eslintrc.json:10]: Check EOF here please.

File: docs/app/globals.css [hunk 2]
```diff
@@ -1,3 +1,4 @@
+@import "./themes.css";
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
```
Comment (MarioGranada) [inline docs/app/globals.css:4]: Use this format please 

```
@import "tailwindcss/base";
@import "tailwindcss/components";
@import "tailwindcss/utilities";
```
Also, add `@import "./themes.css";` at the end.

Comment (MarioGranada) [inline docs/app/globals.css:4]: Moon base css is missing, fonts too.

File: docs/public/stories/412.101a744c.iframe.bundle.js
Comment (MarioGranada) [inline docs/public/stories/412.101a744c.iframe.bundle.js:1]: Add docs/public/stories files to gitignore.

**Review by MarioGranada — CHANGES_REQUESTED** [review]

File: docs/app/globals.css (hunk 2 above)
Comment (robb91-dev) [reply docs/app/globals.css:4]: Could you please explain why?

File: docs/.eslintrc.json (hunk 1 above)
Comment (robb91-dev) [reply docs/.eslintrc.json]: Done

File: docs/public/stories/412.101a744c.iframe.bundle.js
Comment (robb91-dev) [reply docs/public/stories/412.101a744c.iframe.bundle.js:1]: Done

**Pull request merged**